airflow/config_templates/default_airflow.cfg

```diff
@@ -658,6 +658,16 @@ auto_refresh_interval = 3
 # Boolean for displaying warning for publicly viewable deployment
 warn_deployment_exposure = True
 
+# Types of events to ignore for dag audit log view.
+# All other events will be added minus the ones passed here.
+excluded_events = gantt,landing_times,tries,duration,calendar,graph,tree_data
```

Comment:
Can you add this information to config.yml as well?

https://github.com/apache/airflow/blob/14ee831c7ad767e31a3aeccf3edbc519b3b8c923/airflow/config_templates/config.yml#L958-L961

This is what will populate the documentation. 

Also - thoughts on a more descriptive / specific name for these? Something like `audit_view_excluded_events`?

Reply:
+1 on a more descriptive name